src/subscriptions/model/Subscription.ts

```diff
@@ -0,0 +1,14 @@
+import ChannelApeApiError from '../../../src/model/ChannelApeApiError';
+
+export default interface Subscription {
```

Comment:
I'm not sure about making all of these members optional but it doesn't really matter when we are only implementing subsctions.get() we can revisit if we ever expose a subscription update resource (right now you can't update or create your subscription through the API).